scripts/workspace-conversion/workspace-to-dataset.py

```diff
@@ -226,9 +224,9 @@ def get_workspace_contributors(wsAttributes):
         if "library:datasetDepositor" in wsAttributes:
             contributor["contactName"] = wsAttributes["library:datasetDepositor"]
             contributor["correspondingContributor"] = True
-        contributor["email"] = wsAttributes.get("library:contactEmail", None)
-        contributor["intstitution"] = next(
-            iter(wsAttributes.get("library:institute", {}).get("items", [])), None
+        contributor["email"] = wsAttributes.pop("library:contactEmail", None)
```

Comment:
The move to pop is to support unused attributes.

Reply:
If I am understanding correctly, you replace the use of `get` with `pop` so you can see what remains in (the "unused attributes") and show it to the user. Is that correct?